examples/required-options/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import { Base } from "../../index.js";
+export { Base } from "../../index.js";
```

Comment:
Actually I'm not sure if the export is necessary, as we import `MyBase` in the `index.test-d.ts` file?

Reply:
Yeah, I don't see any need to keep this.